### ISSUE TITLE
support hls ?

### ISSUE THREAD

**Issue by dachengai** [clangen/musikcube#106]

**Comment by clangen** [comment]
Yep, HLS support is planned. The idea is to make it an `IDataStream` plugin. Recent releases have added most/all of the required infrastructure. Hopefully I can get to it soon.
